### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by vehbiu** [vercel/community#4005] [Account Issue] [answered]
### Question

Hi,
  First time (iirc) trying to use Vercel. It just says 'This user account is blocked.' when I try to sign up. Anyone know the problem?

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by vehbiu** [comment]
Hey, didn't get an email or anything. So you guys just aren't going to let
my account on for some reason?

**Reply by amyegan** [reply, marked as answer]
I don't have information regarding your account. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a new ticket with the team than can help with account issues 🙏

**Reply by vehbiu** [reply]
Oh, I read the wrong one. My bad...
